src/app/containers/Paragraph/__snapshots__/index.test.jsx.snap

```diff
@@ -1,6 +1,187 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ParagraphContainer should render correctly 1`] = `
+exports[`A react-testing-library snapshot test Should render a ParagraphContainer 1`] = `
+.c0 {
+  color: #404040;
+  font-family: ReithSansNewsRegular,Helvetica,Arial,sans-serif;
+  padding-bottom: 1rem;
+  margin: 0;
+  font-size: 0.9375em;
+  line-height: 1.25rem;
+}
```

Comment:
Looks like if we compare the actual rendered snapshot for `react-testing-library` vs `enzyme`, the former contains the CSS output and the latter does not. I can't see any other differences - does that sound about right?

Reply:
Correct 👍